docs/dapp/tools/localnet.mdx

```diff
@@ -195,7 +196,7 @@ jobs:
           - 8545:8545
           - 8546:8546
         env:
-          OASIS_DEPOSIT_BINARY: /oasis-deposit -n 2
+          OASIS_DOCKER_START_EXPLORER: no
```

Comment:
This makes me wonder, how can one provide -to and -n parameters in the CI after we migrated to the Oasis CLI?

Reply:
:man_shrugging: